src/commonMain/kotlin/io/github/detekt/sarif4k/Merging.kt

```diff
@@ -23,10 +29,18 @@ fun SarifSchema210.merge(other: SarifSchema210): SarifSchema210 {
     )
 }
 
-fun PropertyBag?.merge(other: PropertyBag?): PropertyBag? {
-    return (this?.tags.orEmpty() + other?.tags.orEmpty())
-        .takeIf { it.isNotEmpty() }
-        ?.let { PropertyBag(it) }
+fun PropertyBag.merge(other: PropertyBag): PropertyBag {
+    var merge = this + other
```

Comment:
This will override if two keys exist with the same value. It's not consistent with tags getting properly merged, if we think of "tags" as one of the keys as well.

I don't think you should change anything, just wanted to call this out.

Reply:
Yes in case of collision "other" wins except on `tags` that we handle them.

This repo doesn't have a lot of tests. We should work on that because few test call to few tests. I mean, in detekt I would not dare to add something like this with this coverage.